### PR TITLE
flush only the current workspace by default, --all for all

### PR DESCRIPTION
**Pull request by alxpereira**
# 🚀 Pull Request Overview

### What does this PR do? 🤔

allowing user to flush only the current workspace by default, and using --all if needed all flush
avoiding loss of progression in the other workspaces

## 💻 Changes Summary

- ✅ **Change 1**: Change in the flush method

### CODE REVIEW

File: src/utils/conf.ts [hunk 1]
```diff
@@ -123,10 +123,19 @@ export function addAgent(newAgent: AgentConfig): void {
 /**
  * Clears all agents from the configuration.
```
Comment (shide1989) [inline src/utils/conf.ts:124]: maybe update the JSdoc ?

**Review by shide1989 — COMMENTED** [review]
aside from maybe updating the JSDoc lgtm

**Review by BonnardValentin — APPROVED** [review]

File: src/utils/conf.ts [hunk 2]
```diff
@@ -123,10 +123,20 @@ export function addAgent(newAgent: AgentConfig): void {
...
  */
-export async function flushAgents(): Promise<void> {
+export async function flushAgents(
+  workspaceUrl: string,
+  all?: boolean
+): Promise<void> {
   try {
     const config = readConfig();
-    if (config) {
+    console.log('config, all:', config);
```
Comment (maxbeworks) [inline src/utils/conf.ts]: remove console log

**Review by maxbeworks — CHANGES_REQUESTED** [review]

**Review by BonnardValentin — APPROVED** [review]

**Review by maxbeworks — APPROVED** [review]

File: src/utils/conf.ts (hunk 1 above)
Comment (zhuk-aa) [reply src/utils/conf.ts]: fixed

**Pull request merged**